apps/mobile/src/store/keychains/bitcoin/bitcoin-keychains.write.ts

```diff
@@ -32,6 +32,12 @@ export const bitcoinKeychainSlice = createSlice({
         handleEntityActionWith(adapter.addMany, payload => payload.withKeychains.bitcoin)
       )
 
+      // NOTE: need to adapt this to add multiple accounts
```

Comment:
Curious about this comment. This adapter already uses `addMany`, not `addOne`

Reply:
I added a new action `userAddsAccounts` to add many accounts at once so I think I need to adapt this to also handle their keychains as that's not working fully and the accounts seem incomplete. 

I am hitting issues with undefined `keyOrigin` so I think I missed some things during account creation

Reply:
Solved this. I'm passing an array of account creation data so needed a `flatMap` here